endpoints.json

```diff
@@ -6,22 +6,125 @@
-          "created_at": 1527695953341
+          "article_id": 3,
+          "title": "Eight pug gifs that remind me of mitch",
+          "topic": "mitch",
+          "author": "icellusedkars",
+          "body": "some gifs",
+          "created_at": "2020-11-03T09:12:00.000Z",
+          "votes": 0,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          "comment_count": "2"
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id" :{
+    "description": "serves an individual article by article_id passed into the parametric endpoint",
+    "queries": [],
+    "exampleResponse": {
+      "article": 
+      {
+        "article_id": 3,
+        "title": "Eight pug gifs that remind me of mitch",
+        "topic": "mitch",
+        "author": "icellusedkars",
+        "body": "some gifs",
+        "created_at": "2020-11-03T09:12:00.000Z",
+        "votes": 0,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        "comment_count": "2"
+      }
+    }
+  },
+  "GET /api/articles/:article_id/comments" : {
+    "description": "serves an array of all comments",
+    "queries": [],
+    "exampleResponse": 
+    {
+      "comments": [
+        {
+          "comment_id": 11,
+          "body": "Ambidextrous marsupial",
+          "article_id": 3,
+          "author": "icellusedkars",
+          "votes": 0,
+          "created_at": "2020-09-19T23:10:00.000Z"
+        },
+        {
+          "comment_id": 10,
+          "body": "git push origin master",
+          "article_id": 3,
+          "author": "icellusedkars",
+          "votes": 0,
+          "created_at": "2020-06-20T07:24:00.000Z"
+        }
+      ]
+    }
+    
+  },
+  "POST /api/articles/:article_id/comments" : {
+    "description": "Posts a new comment to the database and serves the added comment",
+    "queries": [],
+    "exampleResponse": 
+    {
+      "comment": {
+        "comment_id": 19,
+        "body": "lorem ipsum",
+        "article_id": 1,
+        "author": "rogersop",
+        "votes": 0,
+        "created_at": "2023-01-19T17:42:39.497Z"
+      }
+    }
+  },
+  "PATCH /api/articles/:article_id" : {
+    "description": "Updates an existing article by the votes property and then serves the single updated article. Example request: { inc_votes: 10 }",
+    "queries": [],
+    "exampleResponse": 
+    {
+      "article": {
+        "article_id": 1,
+        "title": "Living in the shadow of a great man",
+        "topic": "mitch",
+        "author": "butter_bridge",
+        "body": "I find this existence challenging",
+        "created_at": "2020-07-09T20:11:00.000Z",
+        "votes": 101,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      }
+    }
+  },
+  "GET /api/users" : {
```

Comment:
These two end points will need to have an exampleBody, which showed the expected data needed to actually post/patch something i.e. for patch you would expect an object with a single key value pair of inc_votes with any number as the value.